vscode-extensions/debug-certificate-manager-vscode-extension/src/extension.ts

```diff
@@ -139,7 +139,7 @@ export function activate(context: vscode.ExtensionContext): void {
   }
 
   async function handleShowSettings(): Promise<void> {
-    await vscode.commands.executeCommand(VSCODE_COMMAND_WORKSPACE_OPEN_SETTINGS, `@ext:${EXTENSION_ID}`);
+    await vscode.commands.executeCommand(VSCODE_COMMAND_WORKSPACE_OPEN_SETTINGS, `${EXTENSION_ID}`);
```

Comment:
```suggestion
    await vscode.commands.executeCommand(VSCODE_COMMAND_WORKSPACE_OPEN_SETTINGS, EXTENSION_ID);
```
Once you aren't appending anything, may as well use the constant directly.